### PR TITLE
Education Feedback resource outcome

### PR DESCRIPTION
**Pull request by cristobal-buenrostro**

### CODE REVIEW

File: changelog/Microsoft.EducationAssignment.json [hunk 1]
```diff
@@ -1,5 +1,47 @@
...
+          "ChangeType": "Addition",
+          "Description": "Added the **educationFeedbackResourceOutcomeStatus** enumeration type.",
+          "Target": "educationFeedbackResourceOutcomeStatus"
+        },
+        {
+          "Id": "0089a052-0ffa-47ba-a8c0-9fad7f1ae0cc",
+          "ApiChange": "Property",
+          "ChangedApiName": "feedbackResourcesFolderUrl",
+          "ChangeType": "Addition",
+          "Description": "Added the **feedbackResourcesFolderUrl** property to the [educationAssignment](https://docs.microsoft.com/en-us/graph/api/resources/educationAssignment?view=graph-rest-beta) resource.",
```
Comment (JarbasHorst) [inline changelog/Microsoft.EducationAssignment.json:18]: This property hasn't been added yet to the educationAssignment resource. Please, apply this change to the educationAssignment resource or remove this entry from this changelog record.

File: changelog/Microsoft.EducationAssignment.json [hunk 2]
```diff
@@ -1,5 +1,47 @@
...
+          "ChangeType": "Addition",
+          "Description": "Added the [educationFeedbackResourceOutcome](https://docs.microsoft.com/en-us/graph/api/resources/educationFeedbackResourceOutcome?view=graph-rest-beta) resource",
+          "Target": "educationFeedbackResourceOutcome"
+        },
+        {
+          "Id": "0089a052-0ffa-47ba-a8c0-9fad7f1ae0cc",
+          "ApiChange": "Method",
+          "ChangedApiName": "setUpFeedbackResourcesFolder",
+          "ChangeType": "Addition",
+          "Description": "Added the [setUpFeedbackResourcesFolder](https://docs.microsoft.com/en-us/graph/api/setUpFeedbackResourcesFolder?view=graph-rest-beta) method to the [setUpFeedbackResourcesFolder](https://docs.microsoft.com/en-us/graph/api/setUpFeedbackResourcesFolder?view=graph-rest-beta) resource.",
```
Comment (JarbasHorst) [inline changelog/Microsoft.EducationAssignment.json:34]: Please, fix this description and the value for the target property. The correct resource type and target should be: educationAssignment

**Comment by JarbasHorst** [comment]
Hello @cristobal-buenrostro, I reviewed your PR and left some comments. Please, take a look when you will have time. 

To avoid merge conflicts, please merge PR [16915](https://github.com/microsoftgraph/microsoft-graph-docs/pull/16915) into your branch to include my fixes and before addressing my comments.

Also, can you please share with me a pointer to the CSDL and api.md files for this PR?

File: changelog/Microsoft.EducationAssignment.json (hunk 1 above)
Comment (cristobal-buenrostro) [reply changelog/Microsoft.EducationAssignment.json:18]: added to educationAssignment resource

File: api-reference/beta/api/educationsubmission-post-feedbackresource.md [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+doc_type: apiPageType
+---
+
+# Create educationFeedbackResourceOutcome
+
+Namespace: microsoft.graph
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+Add an [educationFeedbackResourceOutcome](../resources/educationfeedbackresourceoutcome.md) to a submission outcome list.
```
[the remark this answers is not in the document]
Comment (cristobal-buenrostro) [reply api-reference/beta/api/educationsubmission-post-feedbackresource.md:16]: done

**Comment by JarbasHorst** [comment]
Hello @cristobal-buenrostro, to continue with the content review work for your PR, I need the CSDL and api.md files. Can you please share with me a pointer to the CSDL and api.md files for this PR?

**Comment by cristobal-buenrostro** [comment]
@JarbasHorst 
https://msazure.visualstudio.com/One/_git/AD-AggregatorService-Workloads/pullrequest/6047914

**Comment by JarbasHorst** [comment]
Hi @cristobal-buenrostro, thanks for addressing my previous comments. I left additional ones. Please, take a look when you will have time.

File: api-reference/beta/resources/educationfeedbackresourceoutcome.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+Represents feedback on an [educationOutcome](educationoutcome.md) object in the form of a document.
+
+## Methods
+
+| Method       | Return Type | Description |
+|:-------------|:------------|:------------|
+| [Set up assignment feedback resources folder](../api/educationassignment-setupfeedbackresourcesfolder.md) | [educationAssignment](educationassignment.md) | Create a SharePoint folder to upload feedback files for a given [educationSubmission](../resources/educationsubmission.md). |
```
[the remark this answers is not in the document]
Comment (cristobal-buenrostro) [reply api-reference/beta/resources/educationfeedbackresourceoutcome.md:22]: removed

File: api-reference/beta/api/educationsubmission-post-feedbackresource.md [hunk 5]
```diff
@@ -0,0 +1,141 @@
+---
```
[the remark this answers is not in the document]
Comment (cristobal-buenrostro) [reply api-reference/beta/api/educationsubmission-post-feedbackresource.md]: done

File: api-reference/beta/api/educationsubmissionfeedbackresource-delete.md [hunk 6]
```diff
@@ -0,0 +1,79 @@
+---
```
[the remark this answers is not in the document]
Comment (cristobal-buenrostro) [reply api-reference/beta/api/educationsubmissionfeedbackresource-delete.md]: done

File: api-reference/beta/resources/educationfeedbackresourceoutcome.md [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+| [Create educationFeedbackResourceOutcome](../api/educationfeedbackresourceoutcome-post-outcomes.md) | [educationOutcome](educationoutcome.md) | Create a new [feedback resource](../resources/educationfeedbackresourceoutcome.md) for a submission. |
+| [Delete educationFeedbackResourceOutcome](../api/educationfeedbackresourceoutcome-delete.md) | None | Delete a [feedback resource](../resources/educationfeedbackresourceoutcome.md) from a submission. |
+
+## Properties
+
+| Property     | Type        | Description |
+|:-------------|:------------|:------------|
+|feedbackResource|[educationResource](educationresource.md)|The actual feedback resource.|
+|id|String|Unique identifier for the **educationFeedbackResourceOutcome**.|
+|resourceStatus|educationFeedbackResourceOutcomeStatus|The possible values are: `notPublished`, `pendingPublish`, `published`, `failedPublish`, and `unknownFutureValue`.|
```
Comment (JarbasHorst) [inline api-reference/beta/resources/educationfeedbackresourceoutcome.md:31]: Please, provide a description for this property.

**Comment by JarbasHorst** [comment]
@cristobal-buenrostro, I added one new comment. Please, take a look when you will have time.

@Lauragra this PR is ready for final review.

File: api-reference/beta/resources/educationfeedbackresourceoutcome.md (hunk 7 above)
Comment (cristobal-buenrostro) [reply api-reference/beta/resources/educationfeedbackresourceoutcome.md:31]: done

**Review by Lauragra — APPROVED** [review]

**Pull request merged**